readthedocs/projects/admin.py

```diff
@@ -180,8 +181,26 @@ class DomainAdmin(admin.ModelAdmin):
     model = Domain
 
 
+class FeatureAdmin(admin.ModelAdmin):
+    model = Feature
+    form = FeatureForm
+    list_display = ('feature', 'projects_list', 'default_true')
+    search_fields = ('feature',)
+    filter_horizontal = ('projects',)
+    readonly_fields = ('add_date',)
+
+    def projects_list(self, feature):
+        projects = [p.name for p in feature.projects.all()[0:10]]
+        if feature.projects.count() > 10:
+            projects += [
+                '... and {0} more'.format(feature.projects.count() - 10)
+            ]
```

Comment:
Surprised this doesn't get done automatically. Does it output the whole list?

Reply:
This view probably isn't useful in this view anyhow. I'll just simplify and make it a count.